addons/respawn/functions/fnc_showFriendlyFireMessage.sqf

```diff
@@ -1,25 +1,35 @@
+#include "script_component.hpp"
 /*
  * Author: commy2
  * Shows a message in system chat of who killed whom.
  *
  * Arguments:
- * 0: Unitn <OBJECT>
+ * 0: Unit <OBJECT>
  * 1: Killer <OBJECT>
  *
  * Return Value:
  * None
  *
  * Example:
- * [ACE_Player, killer] call ace_module_fnc_functionName
+ * [ACE_Player, killer] call ace_respawn_fnc_showFriendlyFireMessage
  *
  * Public: No
  */
-#include "script_component.hpp"
-
 params ["_unit", "_killer"];
 
+// fix, because ace medical cause _unit == _killer
+if (_unit == _killer) then {
+   _killer = _unit getVariable [QEGVAR(medical,lastDamageSource), _killer];
+};
+
 if (_unit != _killer && {side group _unit in [side group ACE_player, civilian]} && {side group _killer == side group ACE_player}) then {
-    systemChat format ["%1 was killed by %2", [_unit, false, true] call EFUNC(common,getName), [_killer, false, true] call EFUNC(common,getName)];
+    private _msg = format ["[FriendlyFire] " + localize LSTRING(FriendlyFire_Message), 
+            ([_unit, false, true] call EFUNC(common,getName)) + ([" (AI)", ""] select (isPlayer _unit)),
+            ([_killer, false, true] call EFUNC(common,getName)) + ([" (AI)", ""] select (isPlayer _killer))
+        ];
+    
+    systemChat _msg;
+    diag_log text _msg;
 
     // Raise ACE globalEvent
     ["ace_killedByFriendly", [_unit, _killer]] call CBA_fnc_globalEvent;
```

Comment:
Nothing you changed, but this functions is called by a global event, and raises another global event